src/todt.d

```diff
@@ -1505,7 +1505,7 @@ extern (C++) class TypeInfoDtVisitor : Visitor
             Parameter arg = (*tu.arguments)[i];
 
             genTypeInfo(arg.type, null);
-            dtxoffDsymbol(dtb, arg.type.vtinfo, 0, dataSymbolRefs);
+            dtxoffDsymbol(dtbargs, arg.type.vtinfo, 0, dataSymbolRefs);
```

Comment:
The problem now is that dataSymbolRefs has dll relocations for "dtb" and not for "dtbargs". I will look for a repro case to make this fail in when using the phobos dll.